src/plugins/fs/Makefile.am

```diff
@@ -7,21 +7,23 @@ libbd_fs_la_LIBADD   = ${builddir}/../../utils/libbd_utils.la ${builddir}/../lib
 libbd_fs_la_LDFLAGS	 = -L${srcdir}/../../utils/ -version-info 2:0:0 -Wl,--no-undefined
 libbd_fs_la_CPPFLAGS = -I${builddir}/../../../include/ -I${srcdir}/../
 libbd_fs_la_SOURCES  = ../check_deps.c ../check_deps.h \
-						../fs.c   ../fs.h   \
-						common.c  common.h  \
-						ext.c     ext.h     \
-						generic.c generic.h \
-						mount.c   mount.h   \
-						ntfs.c    ntfs.h    \
-						vfat.c    vfat.h    \
-						xfs.c     xfs.h     \
-						f2fs.c    f2fs.h
+						../fs.c    ../fs.h    \
+						common.c   common.h   \
+						ext.c      ext.h      \
+						generic.c  generic.h  \
+						mount.c    mount.h    \
+						ntfs.c     ntfs.h     \
+						vfat.c     vfat.h     \
+						xfs.c      xfs.h      \
+						f2fs.c     f2fs.h	  \
```

Comment:
Might as well align the `\`

Reply:
Betryed by my editor, it look aligned (one hidden tab vs. spaces).